tools/nni_trial_tool/trial_keeper.py

```diff
@@ -159,6 +220,8 @@ def check_version(args):
         exit(1)
     check_version(args)
     try:
+        if is_multi_phase():
+            fetch_parameter_file(args)
```

Comment:
we can keep this logic for now, but it is not efficient. A possible alternative is to check hdfs only when received get_next_parameter.

Reply:
agree, let's keep this logic for now since it is easy to implement.